pwndbg/gdblib/next.py

```diff
@@ -104,7 +102,7 @@ def next_matching_until_branch(address=None, mnemonic=None, op_str=None):
         if mnemonic_match and op_str_match:
             return ins
 
-        if set(ins.groups) & jumps:
+        if ins.groups_set & ALL_JUMP_GROUPS:
```

Comment:
Considering this is called like 3 times in this file I wonder if we should make a property out of it or cache it...

Reply:
I think that caching it as a property would be a good call. I'll look to add some caching decorators on the getter functions of the PwndbgInstruction when we start hammering down on the performance.